app/components/public/session-item.hbs

```diff
@@ -1,31 +1,36 @@
 {{!-- template-lint-disable no-nested-interactive  --}}
 <UiAccordion class="segment p-0" style={{css background-color=@session.track.color border-color='lightgray'}}>
-  <div class="title p-4" {{action 'hideSpeakerImage'}} role="button" style={{css color=(text-color @session.track.color)}}>
+  <div class="title p-4 pr-1" {{action 'hideSpeakerImage'}} role="button" style={{css color=(text-color @session.track.color)}}>
     <div class="ui">
-      <h3 class="ui header" id="session-id-{{@session.id}}" style={{css color=(text-color @session.track.color)}}>
-        {{@session.title}}
-        <span style={{css float='right'}}>
+      <h3 class="ui header d-flex space-between" id="session-id-{{@session.id}}" style={{css color=(text-color @session.track.color)}}>
+        <div>
+          {{@session.title}}
+          <div class="sub header" style={{css color=(text-color @session.track.color) opacity=0.6}}>
+            {{@session.sessionType.name}}
+          </div>
+        </div>
+        <div class="d-flex">
           {{#if @session.slidesUrl}}
-            <button class="ui basic {{text-color @session.track.color 'basic' 'inverted'}} button" style={{css color=(text-color @session.track.color 'grey' 'lightgrey')}} {{action this.goToSlides}}>
-              <i class="icon {{if this.slidesUploaded 'download' 'linkify'}}"></i>
-              {{if this.slidesUploaded (t 'Download Slides') (t 'Link to Slides')}}
-            </button>
+            <div style="white-space: nowrap;">
```

Comment:
Why wrap in this?

Reply:
when title is long it break into another line and there is a break down in button text also.
![button](https://user-images.githubusercontent.com/72552281/100738504-aa2fcf80-33fb-11eb-9f8c-1e0b2b38c8c8.png)

to avoid this i wrap this button and put style="white-space: nowrap;" so that there is no line break in button text when title is long.